weldx/util.py

```diff
@@ -421,6 +421,24 @@ def pandas_time_delta_to_quantity(
     return Q_(nanoseconds, "ns").to(unit)
 
 
+def equal_unit_dimensionality(unit_1, unit_2) -> bool:
+    """Return `True` if two units have the same dimensionality and `False` otherwise.
+
+    Parameters
+    ----------
+    unit_1 :
+        First unit
+    unit_2 :
+        Second unit
+
+    Returns
+    -------
+        `True` if both units have the same dimensionality and `False` otherwise.
+
+    """
+    return Q_(1, unit_1).dimensionality == Q_(1, unit_2).dimensionality
```

Comment:
If you know a way to do this without creating a dummy quantity, feel free to make a suggestion

Reply:
There is UnitRegistry.get_dimensionality(input_units) returning a dict base dimensions. This can be checked for quality, but I guess for just two units creating the actual Unit is an easy and readable way.

Reply:
this seems to be implemented with
```python
from weldx.constants import WELDX_UNIT_REGISTRY as ureg
ureg.is_compatible_with("m*s","inch*min")
```

Reply:
Didn't find that one when searching for it. I added it to our [Wiki page about quantities](https://github.com/BAMWelDX/weldx/wiki/Quantities-(pint)) and removed the utility function.